android/SmartLock/app/src/main/java/org/opendds/smartlock/ParticipantLocationListener.java

```diff
@@ -85,7 +86,9 @@ public synchronized void on_data_available(DataReader reader) {
 
         ParticipantLocationBuiltinTopicDataHolder participant =
                 new ParticipantLocationBuiltinTopicDataHolder(
-                        new ParticipantLocationBuiltinTopicData(new byte[16], 0, 0, "", 0, "", 0, "", 0));
+                        new ParticipantLocationBuiltinTopicData(new byte[16],0,0,"",
+                                new Time_t(),"",new Time_t(), "", new Time_t(), "",
```

Comment:
@iguessthislldo should these be new DDS.Time_t() rather than new Time_t()?

Reply:
This is what I had locally.  
                        new ParticipantLocationBuiltinTopicData(new byte[16], 0, 0, "", new DDS.Time_t() ,"", new DDS.Time_t(), "", new DDS.Time_t()));
        SampleInfoHolder si = new SampleInfoHolder(new SampleInfo(0, 0, 0,
                new DDS.Time_t(), 0, 0, 0, 0, 0, 0, 0, false, 0));

Reply:
So I would agree with Fred.

Reply:
Time_t vs. DDS.Time_t should just be an issue of what's `import`ed.  I don't have a preference on this particular issue.  There were definitely recent changes in OpenDDS master that extended this structure to cover IPv6 addresses as well as IPv4.